### PR TITLE
Primitive stuffs

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
This PR is a QoL for primitives; It has a few small features.
First feature being a griddle, I felt it was odd that they had a nice stone oven, nice stove but used just a raging fire for everything else. Second feature is adding the ability to craft Paxil's beautiful wooden fences and gates from wood(Sturdy wooden fence in menu) and allowing you to craft stone cooking objects from stone blocks. The third feature is storage for seeds and grown food, so there is less clutter around the primitive camps, they cannot however be upgraded.

## How This Contributes To The Skyrat Roleplay Experience
First feature is thematic and allows the tribals to cook eggs and pancakes, should they choose.
Second feature is one requested by a handful of people because they like the fences Paxil created, meaning you can repair the camp or expand.
Third feature is to prevent dirty food, clean up clutter and reduce lag caused by massive piles of food. (This is an issue I have been bwoinked for a few times)

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image

</details>

## Changelog

:cl:
add: Added a stone griddle
add: Added a seed shelf & produce bin
add: Crafting recipes for stone cooking structures.
add: Crafting recipes for Paxil's fences.
/:cl:

### CODE REVIEW

**Comment by xXPawnStarrXx** [comment]
I gotta work on getting the emissives on the griddle to work, but they're in the sprite file already. I wanted to get this up first.

File: modular_skyrat/modules/primitive_structures/code/storage_structures.dm [hunk 1]
```diff
@@ -50,3 +50,77 @@
...
+	. = ""
+	if(anchored)
+		. += span_info("It's currently anchored to the floor. It can be [EXAMINE_HINT("wrenched")] loose.")
+	else
+		. += span_info("It's not anchored to the floor. It can be [EXAMINE_HINT("wrenched")] down.")
+	. += span_info("The whole rack can be [EXAMINE_HINT("pried")] apart.")
+/obj/machinery/smartfridge/producebin/welder_act(mob/living/user, obj/item/tool)
+/obj/machinery/smartfridge/producebin/welder_act_secondary(mob/living/user, obj/item/tool)
+/obj/machinery/smartfridge/producebin/default_deconstruction_screwdriver()
+/obj/machinery/smartfridge/producebin/exchange_parts()
```
Comment (jjpark-kb) [inline modular_skyrat/modules/primitive_structures/code/storage_structures.dm]: whats happening here?

File: modular_skyrat/modules/primitive_structures/code/storage_structures.dm [hunk 2]
```diff
@@ -50,3 +50,77 @@
...
+	. = ""
+	if(anchored)
+		. += span_info("It's currently anchored to the floor. It can be [EXAMINE_HINT("wrenched")] loose.")
+	else
+		. += span_info("It's not anchored to the floor. It can be [EXAMINE_HINT("wrenched")] down.")
+	. += span_info("The whole rack can be [EXAMINE_HINT("pried")] apart.")
+/obj/machinery/smartfridge/seedshelf/welder_act(mob/living/user, obj/item/tool)
+/obj/machinery/smartfridge/seedshelf/welder_act_secondary(mob/living/user, obj/item/tool)
+/obj/machinery/smartfridge/seedshelf/default_deconstruction_screwdriver()
+/obj/machinery/smartfridge/seedshelf/exchange_parts()
```
Comment (jjpark-kb) [inline modular_skyrat/modules/primitive_structures/code/storage_structures.dm]: same thing.

Comment (xXPawnStarrXx) [reply modular_skyrat/modules/primitive_structures/code/storage_structures.dm]: It's a mistake, remainder from when it was a copy of a drying rack. I forgot to upload the new version before I went to bed.

**Review by jjpark-kb — APPROVED** [review]

**Pull request merged**